networkx/classes/tests/test_graph.py

```diff
@@ -683,6 +683,9 @@ def test_add_edge(self):
         G = self.Graph()
         G.add_edge(*(0, 1))
         assert G.adj == {0: {1: {}}, 1: {0: {}}}
+        G = self.Graph()
+        with pytest.raises(ValueError):
+            G.add_edge(None, "anything")
```

Comment:
thanks @Qudirah!

Could you also add this test to the `add_edges_from` function too? That will cover all the other untested code in graph.py.

Reply:
done @MridulS